packages/agoric-cli/src/publish.js

```diff
@@ -42,6 +42,17 @@
 
 const { details: X, quote: q } = assert;
 
+/**
+ * @template T
+ * @param {Array<T>} array
+ * @returns {T}
+ */
+const choose = array => {
+  assert(array.length > 0);
+  const index = Math.floor(array.length * Math.random());
```

Comment:
grumble... ambient authority...

Reply:
Fair, and I can change this such that `random` is a power needed by `makeBundlePublisher`.